cmd/tsconnect/src/app/app.tsx

```diff
@@ -43,8 +43,23 @@ class App extends Component<{}, AppState> {
       )
     }
 
+    const lockedOut = netMap && netMap.lockedOut
+    let lockedOutInstructions
+    if (lockedOut) {
+      lockedOutInstructions = (
+        <div class="container mx-auto px-4 text-center">
+          This instance of Tailscale Connect needs to be signed, due to Tailnet Lock being enabled on this domain.
```

Comment:
Slight preference for putting each of these in `<p>...</p>` tags instead of the `<br/></br/>`.

Reply:
Done! I also ended up using `spaces-between` to get the gap between paragraphs, I think thats what I'm meant to do!